### DISCUSSION TITLE
FR Sections navigate to subview

### DISCUSSION THREAD

**Discussion by filikun** [home-assistant/frontend#19938] [Dashboard] [answered]
These sections from the beta is so welcome! I just miss the ability to set a navigate action to the title of the section so that for example if you have a section called "Hallway" clicking that would make you go to the "Hallway" subview. Section titles with an navigate action could have a ">" after the title to highlight that it is clickable. 

Something like this
![image](https://github.com/home-assistant/frontend/assets/9947929/5880f2b2-1549-4cd3-b698-cab8e0862e12)

**Comment by hugz** [comment]
Hello,

This is exactly why I'm here! Please add a tap action feature to the titles of the new sections. 
For example, add a tap action below the title modification like this:
![2024-07-29_13h53_32](https://github.com/user-attachments/assets/48bc5c46-fe18-4e10-8514-65f6dbbc7530)

The actuel workaround is to use "custom:mushroom-title-card" like my old tablet dashboard

![2024-07-29_13h50_16](https://github.com/user-attachments/assets/e240bf3d-ee90-4ed2-adbe-1bbac94bde20)
![2024-07-29_14h00_53](https://github.com/user-attachments/assets/4892ef2d-7e6b-4011-942d-e2e9f6080f08)

Thanks in advance.

Hugo

PS: Sorry for my bad english

**Comment by hugz** [comment, marked as answer]
now included into 2024.10 ! thx HA !
